styles.css

```diff
@@ -59,19 +65,448 @@ ul li a {
+}
+
+.canopy {
+  font-family: 'Poppins', sans-serif;
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  color: #344563;
+}
+
+.card-title {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0;
+  gap: 12px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0;
+  gap: 12px;
+}
+
+.card-detail {
+  display: flex;
+  padding: 0;
+  align-items: center;
+  gap: 12px;
+}
+
+.daily {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.myself {
+  display: flex;
+  width: 327px;
+  height: 35px;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  align-items: center;
+  color: #172b4d;
+  margin-top: 20px;
+  margin-bottom: 12px;
+}
+
+.grid-container {
+  display: grid;
+  background: #e5e5e5;
+  padding-top: 114px;
+}
+
+.skills {
+  display: flex;
+  gap: 10px;
+}
+
+.skills .skill {
+  display: flex;
+  padding: 4px 12px;
+  background: #ebebff;
+  border-radius: 8px;
+  list-style: none;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  color: #6070ff;
+}
+
+.about-me {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 375px;
+  background: white;
+  border-top-right-radius: 50px;
+  margin-top: 30px;
+  margin-bottom: 60px;
+}
+
+.abt-me {
+  width: 327px;
+  height: 35px;
+  margin: 100px 24px 10px;
+}
+
+.abt-me > h1 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+}
+
+.abt-txt {
+  font-family: 'Poppins', sans-serif;
   font-style: normal;
   font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  width: 327px;
+  height: 147px;
+  color: #344563;
+}
+
+.connect {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+}
+
+.abt-me > ul {
+  display: flex;
+  align-items: center;
+  margin-top: 10px;
+  gap: 10px;
+  list-style: none;
+}
+
+.icons {
+  display: flex;
+  color: #dbd8d7;
+  list-style: none;
+  padding: 8px;
+}
+
+.language {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  bottom: 200px;
+}
+
+.java {
+  display: flex;
+  align-items: center;
+  padding: 15px;
+  width: 303px;
+  border-radius: 8px;
+  background: #f7f7f9;
+  gap: 15px;
+}
+
+.html {
+  display: flex;
+  align-items: center;
+  padding: 15px;
+  border-radius: 8px;
+  background: #f7f7f9;
+  gap: 15px;
+}
+
+.css {
+  display: flex;
+  align-items: center;
+  align-content: space-between;
+  padding: 15px;
+  border-radius: 8px;
+  background: #f7f7f9;
+  gap: 15px;
+}
+
+.lan {
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+}
+
+.list-info {
+  margin-top: 350px;
+  display: flex;
+  flex-direction: column;
+  list-style: none;
+  gap: 40px;
+}
```

Comment:
- [ ] Please adhere to the project requirement and try to make your design as close as possible to the `Figma` design because your last code reviewer mentioned that and requested changes because you hadn't implemented or added the skills **>** icon, as I can see still your arrow is not visible.
- [OPTIONAL] Kindly add the arrow and make your design looks at least 90% of the original, even if your plan does have not a suitable font family` and some width and colour issues. Please look at the below design comparison table to understand things better with the help of visualization. Try to visualize your design with Figma to see the differences and please I would like to suggest you implement the things to get approval.

###  Design Comparison

<table>
<tr>
<th>Your Design</th>
<th>Figma Design</th>
</tr>

<tr>
<td>

![Screenshot from 2023-06-09 21-39-27](https://github.com/Surafels/Portfolio_Mobile/assets/66476329/a8c3ef9f-0638-483e-98c4-a5a9b268aebb)

</td>

<td>

![Screenshot from 2023-06-09 21-39-10](https://github.com/Surafels/Portfolio_Mobile/assets/66476329/e1b33706-eea2-4f7b-bdb5-75a801aadfd4)

</td>
</tr>
</table>

#### Requirement

![image](https://github.com/sinruidotec/Portfolio-part-2-Microverse/assets/66476329/641d0c3f-f77f-4683-a5d1-b0c8780bdbe6)